PR to add Ericsson involvement to the work

rkandoi opened the pull request:
#### What type of PR is this?

Add one of the following kinds:
* repository management

#### What this PR does / why we need it:

Adding Ericsson as CODE OWNER and MAINTAINER for the IoT Network Optimization repository.

rkandoi committed:
PR to add Ericsson involvement to the work

hdamker commented:
Waiting for approval by current @camaraproject/iot-network-optimization_codeowners (just the note that it is not necessary to be codeowner to start contributing).

FabrizioMoggio reviewed (approved):
OK to me

hdamker commented:
Invited @rkandoi to the CAMARA Project organization - waiting until the invite is accepted (and with that the CODEOWNERS file get valid). Please ping me when this is the case.

BTW: for this PR it is not necessary that the PR validation workflow gets green - but if you want to achieve that you would need to sync the PR branch with the main of https://github.com/camaraproject/IoTNetworkOptimization

FabrizioMoggio commented:
@rkandoi please can you proceed as suggested here: https://github.com/camaraproject/IoTNetworkOptimization/issues/32#issuecomment-3461906873

rkandoi commented:
> Invited @rkandoi to the CAMARA Project organization - waiting until the invite is accepted (and with that the CODEOWNERS file get valid). Please ping me when this is the case.
> 
> BTW: for this PR it is not necessary that the PR validation workflow gets green - but if you want to achieve that you would need to sync the PR branch with the main of https://github.com/camaraproject/IoTNetworkOptimization

Thanks @hdamker. I have accepted the invite. I also rebased this on top of the latest. (and updated this comment).

rkandoi committed:
Merge branch 'camaraproject:main' into main

hdamker reviewed (approved):
Teams @camaraproject/iot-network-optimization_codeowners and @camaraproject/iot-network-optimization_maintainers up-to-date, CODEOWNERS file valid -> all done, ready to merge
